### DISCUSSION TITLE
Letters within words are randomly being cut off / incompletely pushed to the next line (Pictures in body of text)

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1697] [Help]
<img width="100" alt="Why are words being cut off pt 2" src="https://github.com/freeplane/freeplane/assets/147225942/3fd18ec2-71ce-454b-a05c-b3704639ba02">
<img width="213" alt="Why are words being cut off" src="https://github.com/freeplane/freeplane/assets/147225942/cdbf8ac5-0e07-4070-9ef8-8a95db551737">

Hey all. I am not sure what's going on or how to fix this, but I have noticed that randomly the letters within words will be split, so that some of the letters are on one line and others are pushed to the next line. What is causing this issue, and how can I remedy this? Thanks!

**Comment by EdoFro** [comment]
are you copy/pasting them from thw web or pdf?

Maybe they come with invisible characters in the word (between "c" and "hooses").

**Reply by EdoFro** [reply]
If so, you can do a find/replace for the whole map to solve the issue.

**Reply by EdoFro** [reply]
can you attach a new mindmap whith a copy of that node?
Look first that the issue is present there too.

**Reply by adamfreeplaneuser01** [reply]
@EdoFro I made a new mind map and retyped everything. It's weird, here the issue only happens if I italicize the word "unjustifiably". Also, to clarify, I am not copying/pasting the text, I typed it all out. 
 
<img width="343" alt="test mind map screenshot" src="https://github.com/freeplane/freeplane/assets/147225942/53e8a2dc-63ec-4ef6-a475-3aec4067c828">

I tried to upload the mind map here but I guess the file type isn't supported on Github

**Comment by adamfreeplaneuser01** [comment]
@EdoFro @dpolivaev Hi, this issue is still persisting. And this texted was not copied/pasted from anywhere, I typed it out 
<img width="364" alt="Screenshot 2024-03-04 113612" src="https://github.com/freeplane/freeplane/assets/147225942/7043f025-7201-498d-be78-93504a7226fc">

**Reply by dpolivaev** [reply]
If you want to upload your mind map file here you can zip it.
I don't think I can fix it because I use libraries for rendering, but I could have a look.

**Reply by adamfreeplaneuser01** [reply]
@dpolivaev Thanks. Hopefully you can open what I attached here. The specific nodes are found under "Fraud and property damage" -> "Fraud Damages" 
[For freeplane github.zip](https://github.com/freeplane/freeplane/files/14489422/For.freeplane.github.zip)
